pkg/prometheus/promcfg.go

```diff
@@ -4185,6 +4203,7 @@ func (cg *ConfigGenerator) generateTracingConfig() (yaml.MapItem, error) {
 	}, nil
 }
 
+// [mviswanathsai]TODO: I don't see any version checks for the ProxyConfig, should we add it?
```

Comment:
the version check is in `addProxyConfigtoYaml()`

Reply:
It's been a while, but I think what I meant was checks in `resource_selector.go`.